src/components/OptionPanel/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { PureComponent } from 'react';
+import './styles.css';
+import { Card, CardBody, CardTitle, CardText, Button, ButtonGroup } from 'reactstrap';
+
+export default class OptionPanel extends PureComponent {
+  getButtons = () =>
+    this.props.options.map((option, i) => this.generateButton(option.label, option.value, option.color, i));
+
+  generateButton = (label, value, color, counter) => (
+    <Button
+      key={counter}
+      color={value === this.props.value ? color : 'secondary'}
+      onClick={() => this.handleClick(value)}>
```

Comment:
I don't think you can avoid this, but just pointing out that this affects performance